chirpstack/src/cmd/configfile.rs

```diff
@@ -618,6 +618,15 @@ pub fn run() {
     # is needed.
     assume_email_verified={{ user_authentication.openid_connect.assume_email_verified }}
 
+    # Add additional scopes.
```

Comment:
Would it be an idea to define this as `scopes`, and set the default value to `["email", "profile"]`? Then we have one config value with all the scopes. What do you think?

Reply:
Yes definitely a good idea! I‘m on vacation since the 18th of Juli. I can do it when I’m back or you can do it if you find time :)

Reply:
Enjoy your vacation. I just pushes some changes to this PR.